src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -509,7 +500,7 @@ bool WholeBodyDynamicsDevice::openExternalWrenchesPorts(os::Searchable& config)
     {
         std::string port_name = outputWrenchPorts[i].port_name;
         outputWrenchPorts[i].output_port = new yarp::os::BufferedPort<yarp::sig::Vector>;
-        ok = ok && outputWrenchPorts[i].output_port->open(port_name);
+        ok = ok && outputWrenchPorts[i].output_port->open(portPrefix+port_name);
```

Comment:
So a boolean to enable the use of portPrefix external wrench ports?
In theory portPrefix will only be used when more than one wholebodydynamics is running and in that case it was simpler to change just the port prefix. Otherwise for avoid port name collision you need to change not only the port prefix but also the individual names for all the external wrench ports. 
Having said that normally people do not launch more than one wholebodydynamics unless they are testing stuff like me.

Reply:
So given that you can fully change the name from the configuration file the boolean option might be worthless. So it would be just reverting to the previous right? That should keep it optional.

Reply:
> 
> 
> So given that you can fully change the name from the configuration file the boolean option might be worthless. So it would be just reverting to the previous right? That should keep it optional.

The problem is that the `portPrefix` option is already used for other ports, so we can't  just change its default value to `""`, otherwise we will change the port name of a lot of other ports. The options are either: 
* have a boolean parameter with a name like "scopedExternalWrenchesPortNames` that enable or disable the use of `portPrefix` 
* have a different string parameter called `externalWrenchesPortPrefix` , that is just use for this port names
both are fine for me, the only thing that it would be not managable is to break compatibility with old port names, as those are already hardcoded in a lot of existing project/configuration files.

Reply:
> The problem is that the portPrefix option is already used for other ports, so we can't just change its default value to "", otherwise we will change the port name of a lot of other ports.

yeah I realized that when I looked in the code and started asking my self if I had introduced those changes as well haha. I think for now I will leave it as it was since is a change that is not related to the functionality of temperature compensation itself. We might open an issue and decide the best way to handle it and if its even necessary from the point of other users.

Reply:
Ok for me.